roles/container-engine/cri-o/vars/centos-7.yml

```diff
@@ -1,5 +1,12 @@
 ---
-default_crio_packages:
-  - cri-o-1.19.0
+crio_versioned_pkg:
+  "1.19":
+    - cri-o-1.19.0-13.2.el7
```

Comment:
Using specific version like this will break when the package is updated in the repo. Just use "cri-o", no need for a specific version as the yum repos are versioned already. Same goes for Debian I think.

Reply:
ohh
repo сontains only the one last built image.